ushahidi/platform-client
# FIX(Location in posts):Validating manual lat-long fields

This pull request makes the following changes:
- Adds validation for lat/long location-fields when creating a post
- Shows and hides the update-map button based on validation

Testing checklist:
-  Create a new post with a location-field
- Write some random text or invalid latitude or longitude in the lat/lon boxes
- [ ] You should see an error-message
- [ ] The map should not update
- [ ] The update Map button should not be visible
- Add correct values to the fields
- [ ] The update Map button should be visible
- Click on the map or search for a location in the map
- [ ] The latitude and longitude in the input-boxes should update accordingly
- [ ] The update Map button should not be visible
- Change the values in the inputboxes again
- [ ] The update Map button should be visible

- [x ] I certify that I ran my checklist

Fixes ushahidi/platform#3634 .

Ping @ushahidi/platform
